cli/js/compiler_util.ts

```diff
@@ -289,8 +289,6 @@ export const ignoredDiagnostics = [
   // TS1103: 'for-await-of' statement is only allowed within an async function
   // or async generator.
   1103,
```

Comment:
No supoort for top-level for-await-of yet?

Reply:
No, I believe technically it is seperate from top level await in TC39 and isn't stage 3 yet. 🤷‍♂